hoomd/SnapshotSystemData.h

```diff
@@ -66,6 +66,9 @@ template<class Real> struct SnapshotSystemData
      */
     void replicate(unsigned int nx, unsigned int ny, unsigned int nz);
 
+    //! Wrap the snapshot
```

Comment:
Be more explicit about the behavior. This is kind of long but there's probably a way to convey all of this information about expected inputs/outputs.
```suggestion
    //! Wrap the snapshot's particle positions into the box. Particle images are updated based on the number of wrapped images.
```